metadata.sh

```diff
@@ -38,7 +38,7 @@ export VENDOR_DIRECTORY="${_project_directory}/shared/vendor"
 export CASSANDRA_VERSION="3.11.5"
 
 # https://github.com/kudobuilder/kudo/releases/tag/vx.y.z
-export KUDO_VERSION="0.11.1"
+export KUDO_VERSION="0.13.0"
```

Comment:
@ANeumann82 can you also please bump in here https://github.com/mesosphere/kudo-cassandra-operator/blob/master/kuttl-tests/Makefile#L1 ?